### PR TITLE
நம்Chain Updated

### PR DESCRIPTION
**Pull request by ramagururadhakrishnan**
Updated about நம்Chain Project to the list of projects that is using Multiformats. Kindly review them and approve.

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@Kubuxu @dignifiedquire @hsanjuan -- could one of you please review? Thank you!

File: content/index.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ The Multiformats project began through [the IPFS Project](https://ipfs.io). It i
 		<img height="64px" src="https://github.com/libra/libra/raw/master/.assets/libra.png" />Libra</a>
 		- Libra’s mission is to enable a simple global currency and financial infrastructure that empowers billions of people.
 	</li>
+	<li>
+		<a href="https://1.bp.blogspot.com/-0SArWfduw68/XkxV8EmBBcI/AAAAAAAAABw/h9aWSWbm0J4kilgn3xddzQ3PdoP-e3RZgCLcBGAsYHQ/s1600/SAVE_20200127_132431.jpg">
```
Comment (hsanjuan) [inline content/index.md:80]: I think this should be linking the project, not the image?

Comment (ramagururadhakrishnan) [reply content/index.md:80]: Thank you. Will update.

Comment (ramagururadhakrishnan) [reply content/index.md:80]: https://github.com/NamChain-Open-Initiative-Research-Lab/NamChain is the link of the project.

Comment (hsanjuan) [reply content/index.md:80]: Can you update the link in the PR

Comment (ramagururadhakrishnan) [reply content/index.md]: Updated.

**Pull request merged**